src/nimgen/c2nim.nim

```diff
@@ -85,11 +85,10 @@ proc c2nim*(fl, outfile: string, c2nimConfig: c2nimConfigObj) =
       passC &= "const header$# = currentSourcePath().splitPath().head & \"$#\"\n" %
         [fname, file.relativePath()]
     extflags = "--header:header$#" % fname
-
   # Run c2nim on generated file
   var cmd = "c2nim $# $# --out:$# $#" % [c2nimConfig.flags, extflags, outfile, cfile]
   when defined(windows):
-    cmd = "cmd /c " & cmd
+    cmd = "cmd /c " & cmd.quoteShell
   discard execProc(cmd)
```

Comment:
Why not quoteshell for everything, not just Windows?

Reply:
Because this will wrap the whole command in quotes, which other platforms don't need and will cause errors. There's a few files in nim-libnx that cause errors if this isn't set on Windows.